Source/DafnyCore/TestGenerationOptions.cs

```diff
@@ -52,11 +53,8 @@ public enum Modes { None, Block, Path };
           }
           return true;
 
-        case "generateTestInlineDepth":
-          var depth = 0;
-          if (ps.GetIntArgument(ref depth)) {
-            TestInlineDepth = (uint)depth;
-          }
+        case "generateTestInline":
```

Comment:
Consider using `DafnyOptions.RegisterLegacyUi` to replace the `generateTestInline` related code in this file.